common/ledger/blockledger/fileledger/factory.go

```diff
@@ -97,8 +109,30 @@ func New(directory string, metricsProvider metrics.Provider) (blockledger.Factor
 	if err != nil {
 		return nil, err
 	}
-	return &fileLedgerFactory{
+
+	fileRepo, err := filerepo.New(filepath.Join(directory, "filerepo"), "remove")
```

Comment:
I'm guessing this is not being introduced here.... but "filerepo" seems almost uselessly vague.  Something like "pendingops" or something more descriptive could be nice.  (No need to change here, but, if agreed, perhaps open a JIRA?)

Reply:
Opened https://jira.hyperledger.org/browse/FAB-18280